SwiftWisdom/Core/StandardLibrary/Numbers/Integer+Extensions.swift

```diff
@@ -56,3 +56,32 @@ extension NumberFormatter {
         return decimalFormatter
     }
 }
+
+extension Integer {
+    fileprivate var ip_data: Data {
```

Comment:
This is more inline with the `var ip_data: Data` on all of the other primitive types